### ISSUE TITLE
Uploading annotations onto a file which already contains annotations

### ISSUE THREAD

**Issue by allorens** [johentsch/ms3#4]
The library allows uploading annotations onto a file which already contains annotations for the same select staff. However and as expected, this turns the file into a corrupt one. What about a warning before the annotations are upladed?

**Comment by allorens** [comment]
Sometimes this happens even when the file did not previously contain any labels. After saving it, if one tries to open it, the following error message appears:
![image](https://user-images.githubusercontent.com/61057980/95877767-bf31a000-0d74-11eb-9a27-b29e9ee848cd.png)


Translation: Error when loading. The file C:/... can't be opened. Bad format.
Details: XML reading error in line 25, column 29: metaTag

This file already had some extra metadata. When reversing the order (1st attach labels and then add metadata) the error didn't occur.

**Comment by johentsch** [comment]
Oh very interesting, could you please send me such a corrupted file? Have you encountered this error multiple times?

**Comment by allorens** [comment]
Yes, it happened every time I followed the same procedure: adding metadata and then uploading annotations.

**Comment by johentsch** [comment]
Hi @allorens, is this error still happening? If yes, could you please paste the XML code around the line indicated in the error message?
